src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2451,6 +2451,8 @@ void processContextElement
   // FIXME P6: Once we allow for ServicePath to be modified, this loop must be looked at.
   //
   int docs = 0;
+  // to check updates
+  bool isUpdate = false;
```

Comment:
If you put the comment on the same line (EOL), you gain a line.
These short comments we try to put on the same line.
Longer comments go on a separate line, of course.

Also, align the variable names and the eq-sign, like this:

    int  docs     = 0;
    bool isUpdate = false;  // to check updates

It's easier to read this way ...